Src/Sarif.PatternMatcher/AnalyzeCommand.cs

```diff
@@ -36,6 +36,12 @@ public class AnalyzeCommand : MultithreadedAnalyzeCommandBase<AnalyzeContext, An
                 SearchDefinitions definitions =
                     JsonConvert.DeserializeObject<SearchDefinitions>(searchDefinitionsText);
 
+                // This would skip files that does not look like rules.
+                if (definitions == null || definitions.Definitions == null)
```

Comment:
>if (definitions == null || definitions.Definitions == null) [](start = 16, length = 59)

We might find files that aren't rules. If that happen, the project fails. With this, we will just continue.